### PR TITLE
docs: add a simple example to the FAQ

### PR DESCRIPTION
**Pull request by chapmanjacobd**
It might be somewhat obvious but I keep having to relearn it

### CODE REVIEW

**Comment by jiegec** [comment]
Please squash your commits into one and format the commit message.

**Comment by chapmanjacobd** [comment]
okay :heavy_check_mark:

File: 00FAQ [hunk 1]
```diff
@@ -2203,8 +2203,15 @@ ______________________________________________________________________
...
-	of those files, or use the lsof +D and +d options.
+	directory, you need to either specify the individual path names
+	of those files, their common mount point, or use the lsof +D 
+	and +d options.
+	
+	For example, if you wanted to check the files open in your /lsof 
+	folder then the following might be the quickest way to get
+	the answer you are looking for:
+
+	lsof / | grep /lsof | grep -v DIR
```
Comment (jiegec) [inline 00FAQ:2214]: Why not `lsof /lsof`?

Comment (chapmanjacobd) [reply 00FAQ:2214]: This section of the FAQ is 

```
3.21.4	Why won't lsof find all the open files in a directory?
```

`lsof /lsof` will give different results from `lsof / | grep /lsof` unless `/lsof` is a mountpoint. 

You would need something like `lsof +D /lsof` to be without `grep` or maybe there is another feature of lsof that I am unaware of which would be similar

Comment (jiegec) [reply 00FAQ:2214]: I see.

**Review by jiegec — APPROVED** [review]
Please add yourself to `00CREDITS` and change log to `00DIST` in a separate commit.

File: 00CREDITS [hunk 2]
```diff
@@ -569,6 +569,7 @@ provided test systems where I was able to do development work.
 	Sergei Trofimovich (@trofi)
 	Danilo Spinella (@danyspin97) and
 	Jiajie Chen (@jiegec)
+	Jacob Chapman (@chapmanjacobd)
```
Comment (jiegec) [inline 00CREDITS:572]: Please move the `and` to Line 571

Comment (jiegec) [reply 00CREDITS:572]: Please format the last commit message as well.

Comment (chapmanjacobd) [reply 00CREDITS:572]: nobody else has an `and`. it likely makes more sense to remove that or place both names on one line. I'm not 100% sure what you want to do

either way I feel like that change is out of scope for this PR. I never modified line 570 or 571. if a previous PR was incorrect it is the maintainer's job to fix it. feel free to add commits to this branch

Comment (jiegec) [reply 00CREDITS:572]: I mean you should change it to:

```
trofi
danyspin97
jiegec and
chapmanjacobd
```

Comment (chapmanjacobd) [reply 00CREDITS:572]: mmmmk
I thought it was a line delimited list

**Comment by jiegec** [comment]
Merged in commit 7a7d1827faaf5fe558f6c0e16861f683b111fd97

**Pull request closed without merging**